packages/editor/src/components/properties/Util.ts

```diff
@@ -64,9 +64,13 @@ export const updateProperties = <C extends Component>(
     : editorState.lockPropertiesPanel.value
     ? [UUIDComponent.getEntityByUUID(editorState.lockPropertiesPanel.value)]
     : SelectionState.getSelectedEntities()
-  for (let i = 0; i < affectedNodes.length; i++) {
-    const node = affectedNodes[i]
-    updateComponent(node, component, properties)
+
+  for (const node of affectedNodes) {
+    try {
+      updateComponent(node, component, properties)
```

Comment:
what is the underlying issue? we should to solve it in the video component and allow it to fail gracefully if possible, rather than just try catching

Reply:
From what I observed, we dont have any error handling within the `updateComponent` function. In the context of this PR, when I add a URL path that is invalid i.e empty or some random text, `updateComponent` will fail which causes the video to stop.

(For some reason I just saw this comment, I apologize for the delay response)

Reply:
We could throw an error and display it on the screen but I think that's more of a design question. Im open for feedback.

Reply:
error handling should happen in the component itself, and we should surface those to the UI of the properties editor. Many components leverage the `ErrorComponent` via `errorFunctions` for this